server/xpub/entities/manuscript/resolvers.js

```diff
@@ -214,7 +214,7 @@ ${err}`,
         }
       } catch (error) {
         logger.warn('Manuscript conversion failed', {
-          error,
+          error: error.message,
```

Comment:
So I am guessing that `error` had the entire file contents in as well?

Reply:
Are there any other members of `error` that might be useful, other than `message`?

Reply:
I think the message is the most relevant. There is also
name: the name of the error (e.g. `StatusCodeError`)
status code: e.g. 500 (this is included in `error.message` as well)
error: the error generated (this is included in `error.message` along with the code)
options: the options/arguments for `request.post`
constructor: the constructor for the error object (the actual code)
response: the response object from `request.post` (including the file)

Reply:
Let's leave it as-is, happy for it to get merged.